solutions/devsprint-firas-safa-1/GitHubApp/Screens/List/ListView.swift

```diff
@@ -12,16 +12,12 @@ struct ListViewConfiguration {
 }
 
 final class ListView: UIView {
-
-    private let listViewCellIdentifier = "ListViewCellIdentifier"
-    private let repositoryCellHeight: CGFloat = 70
-
     private var listItems: [String] = []
 
     private lazy var tableView: UITableView = {
         let tableView = UITableView(frame: .zero)
         tableView.translatesAutoresizingMaskIntoConstraints = false
-        tableView.register(RepositoryCell.self, forCellReuseIdentifier: listViewCellIdentifier)
+        tableView.register(RepositoryCell.self, forCellReuseIdentifier: RepositoryCell.classIdentifier())
```

Comment:
not quite.. create a static variable like we did with the height, and refer to that.